Libraries/Text/TextInput/RCTBaseTextInputView.m

```diff
@@ -527,6 +527,10 @@ - (void)textInputDidReturn
 
 - (NSString *)textInputShouldChangeText:(NSString *)text inRange:(NSRange)range
 {
+  if (text == nil) { // [macOS
+    return nil;
+  } // macOS]
+    
```

Comment:
This seems like a much riskier change since it's now returning before a bunch of code is executed?

Reply:
Yes, this was intentional. According to another comment in this method, returning nil rejects the change, which seems to be the right thing to do. If you'd prefer the original check we can start with that.

Reply:
Reverted back to the original fix